rust/repositories.bzl

```diff
@@ -293,6 +293,8 @@ def _rust_toolchain_repository_impl(ctx):
     for target_triple in [ctx.attr.exec_triple] + ctx.attr.extra_target_triples:
         BUILD_components.append(_load_rust_stdlib(ctx, target_triple))
 
+    BUILD_components.append(_load_rust_stdlib(ctx, "wasm32-unknown-unknown"))
```

Comment:
Apologies if this is made obvious in a comment elsewhere in PR:

What's the motivation for specially including "wasm32-unknown-unknown"'s stdlib and toolchain decl instead of using the existing "extra_target_triples" mechanism?

e.g.
```python
    rust_repository_set(
        name = "rust_linux_x86_64",
        exec_triple = "x86_64-unknown-linux-gnu"
        # Specify that wasm32 should be brought in
        extra_target_triples = ["wasm32-unknown-unknown"],	
        version = RUST_VERSION,
    )
```
If that works, I'd (slightly) prefer that approach rather than have everyone automatically get wasm support. If we change to that, it'd probably be prudent to also mention it in the WASM section of the readme.

Reply:
Good catch. Done.